packages/webapp/src/containers/Finances/CustomRevenueType/constants.js

```diff
@@ -0,0 +1 @@
+export const CUSTOM_REVENUE_NAME = 'custom_revenue_name';
```

Comment:
This needs to be `revenue_name` same as the database for your payload to be correct.

Reply:
Thank you!